README.md

```diff
@@ -25,59 +25,29 @@ The key features are:
 - Sign detection;
 - Safety alerts;
 - Object detection;
-- Lane detection.
+- Lane detection;
+- Support for external cameras.
```

Comment:
I doubt this is a key feature, moreover, there's no out-of-the-box support for any specific camera, just a way to feed arbitrary frames to the SDK.

Reply:
Totally makes sense!

Reply:
Removed.